Server/src/main/java/org/bsu/twiter/servlets/TwitServlet.java

```diff
@@ -0,0 +1,82 @@
+package org.bsu.twiter.servlets;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.bsu.twiter.forms.TwitCreateForm;
+import org.bsu.twiter.forms.TwitUpdateForm;
+import org.bsu.twiter.models.Twit;
+import org.bsu.twiter.services.TwitService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Optional;
+
+@WebServlet("/tweets")
+public class TwitServlet extends HttpServlet {
+
+    private TwitService twitService;
+
+    @Override
+    public void init() throws ServletException {
+        twitService = new TwitService();
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        Long id = Long.parseLong(req.getParameter("id"));
+        Optional<Twit> twitOptional = twitService.getTwit(id);
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Not a mistake, but from a pure logic's standpoint it's better to initialize `ObjectMapper` no sooner than we learned that `twitOptional.isPresent()`. 
It is not needed before that point. 
Just imagine that `ObjectMapper objectMapper = new ObjectMapper();` is a very costly procedure, like establishing Internet connection and downloading  a gb of data)